submissions/NastjonkaK/JS-DOM/Style/style.css

```diff
@@ -0,0 +1,122 @@
+body {
+	font-size: 16px;
+	font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+	background-color: rgb(117, 117, 117);
+	color: #fff;
+}
+
+.container {
+	margin: 5% auto;
+	width: 80%;
+	background-image: linear-gradient(rgba(0,0,0, 0.7), rgba(0,0,0, 0.7)), url("../Sourse/Europe.jpg");
+	background-size: 100%;
+	background-position: center;
+	border: 2px solid #bbb;
+	border-radius: 5px;
+}
+
+.content {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	padding: 5%;
+	padding-bottom: 25%;
+}
+
+.menu {
+	width: 15%;
+}
+
+.countries {
+	padding: 0;
+	height: 100%;
+	display: flex;
+	justify-content: flex-start;
+	flex-direction: column;
+}
+
+.countries li {
+	list-style-type: none;
+	width: 100%;
+	margin-top: 10px;
+	padding: 10%;
+	background-color: #1aaaaa;
+	border-radius: 5px;
+	text-align: center;
+}
+
+.countries li:hover,
+.countries li:active {
```

Comment:
Styles on `:active` state applies when a user make action to interactive elements. If styles on `:hover` and `:active` will be the same how do you tell the user about clicking on that element?)